stdlib/ballerina-http/src/main/ballerina/http/secure_listener.bal

```diff
@@ -122,7 +122,7 @@ function addAuthFiltersForSecureListener (SecureEndpointConfiguration config) {
 @Return {value:"Array of Filters comprising of authn and authz Filters"}
 function createAuthFiltersForSecureListener (SecureEndpointConfiguration config) returns (Filter[]) {
     // parse and create authentication handlers
-    //AuthHandlerRegistry registry = new;
+    AuthHandlerRegistry registry = new;
```

Comment:
Just calling `AuthHandlerRegistry registry;` is sufficient. It is not necessary to new it.